zproject/config.py

```diff
@@ -9,6 +9,7 @@
 
 # Whether this instance of Zulip is running in a production environment.
 PRODUCTION = config_file.has_option("machine", "deploy_type")
+STAGING = config_file.get("machine", "deploy_type", fallback="") == "staging"
```

Comment:
I think we already have a `settings.STAGING` that's set in `default_settings.py`  and overridden in `/etc/zulip/settings.py`, and can just use that here.

Reply:
(Or is the constraint that we don't want to import `settings`?

Reply:
We can't import settings, because this is imported _during_ the load in `zproject/computed_settings.py`:
https://github.com/zulip/zulip/blob/a41d5e58f9b958422acd508ff724c08097ea368b/zproject/computed_settings.py#L1183-L1188

Reply:
We could pass in the value of STAGING, I suppose.

Reply:
OK